docs/docs/production.rst

```diff
@@ -0,0 +1,58 @@
+.. _production:
+
+.. note::
+
+   To use this tutorial, you need to :ref:`install unitxt <install_unitxt>`.
+
+=====================================
+Using in production
+=====================================
+
+Unitxt can be used to process data in production. First define a recipe:
```

Comment:
is production the correct word? i.e. dynamically, "on-the-fly", etc?

Reply:
Yes, the regular use case of unitxt is either in training or evaluation. The use case being addressed here is the use in production time.